### PR TITLE
CFY-5548-add-cli-ref-docs

### PR DESCRIPTION
**Pull request by nir0s**

### CODE REVIEW

File: content/cli/ssh.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+layout: bt_wiki
+title: ssh
+category: Docs
+draft: false
+abstract: Cloudify's Command-Line Interface
+weight: 40
+---
+
+The `cfy ssh` command is used to connect to a Cloudify manager via ssh.
```
Comment (nirfuchs) [inline content/cli/ssh.md]: I suggest that other than the actual 'cfy ssh' command any other mention of ssh in the documentation should be capitalized. (as a reference you can have a look at Vagrant's documenation)

File: content/cli/snapshots.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+Download a snapshot from the manager.
+
+#### Required flags
+
+* `-s, --snapshot-id=SNAPSHOT_ID` - The ID of the snapshot
+
+#### Optional flags
+
+* `-o, --output=OUTPUT_PATH` - The output path for the downloaded file.
```
Comment (nirfuchs) [inline content/cli/snapshots.md]: be consistent with all other descriptions of Optional flags and remove the dot at the end-of-line.

File: content/cli/snapshots.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+
+### upload
+
+Usage: `cfy snapshots upload -p SNAPSHOT_FILE -s SNAPSHOT_ID` 
+
+#### Required flags
+
+* `-s, --snapshot-id=SNAPSHOT_ID` - The ID of the snapshot
+* `-p, --snapshot-path=SNAPSHOT_FILE` - The local path of the snapshot to upload.
```
Comment (nirfuchs) [inline content/cli/snapshots.md]: same comment as above about the dot at end-of-line

File: content/cli/maintenance-mode.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+draft: false
+abstract: Cloudify's Command-Line Interface
+weight: 20
+---
+
+The `cfy maintenance-mode` command is used to restrict REST access to the manager.
+
+This is required, for instance, when upgrading a manager via the `cfy upgrade` or `cfy rollback` commands.
+
+Putting the manager in maintenance mode prevents it from running any executions.
```
Comment (nirfuchs) [inline content/cli/maintenance-mode.md]: should it be maintenance-mode instead of maintenance mode?

File: content/cli/maintenance-mode.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+See [maintenance-mode]({{< relref "manager/maintenance-mode.md" >}}) for more information.
+
+
+## Commands
+
+### activate
+
+Usage: `cfy maintenance-mode activate [options]`
+
+Activate maintenance mode on the manager disallowing further REST requests.
```
Comment (nirfuchs) [inline content/cli/maintenance-mode.md]: should it be maintenance-mode instead of maintenance mode?

File: content/cli/logs.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+#### Required flags
+
+* `-f, --force` - This flag is required to perform the purge.
+
+#### Optional flags
+
+* `--backup-first` - Executes a `cfy logs backup` first.
+
+{{% gsNote title="Warning" %}}
+USE WITH CARE! Log files in Cloudify manager are rotated. This is a safety measure in case the disk space on the manager runs out for some reason.
```
Comment (nirfuchs) [inline content/cli/logs.md]: I think 'the' can be omitted.

**Comment by nir0s** [comment]
Thanks. All valid comments and will be changed.

File: content/cli/agents.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+title: agents
+category: Docs
+draft: false
+abstract: Cloudify's Command-Line Interface
+weight: 10
+---
+
+The `cfy agents` command is used to install Cloudify agents on existing deployments.
+
+In certain situations, you might want to install agents on VMs of deployments in which agents were not previously installed.
```
Comment (dankilman) [inline content/cli/agents.md]: not so correct

File: content/cli/blueprints.md [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+
+*  `-o, --output=OUTPUT` -
+                        The local path of the downloaded blueprint
+
+
+### validate
+
+Usage: `cfy blueprints validate -p BLUEPRINT_PATH` 
+
+Validate a blueprint. This checks that the blueprint's syntax is valid and that all imports are accessible.
```
Comment (dankilman) [inline content/cli/blueprints.md]: it is important to note that it checks import accessibility *client side* only

File: content/cli/install.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
+---
+layout: bt_wiki
+title: install
+category: Docs
+draft: false
+abstract: Cloudify's Command-Line Interface
+weight: 100
+---
+
+The `cfy install` command is used to install an application using a Cloudify manager without going through the process of uploading a blueprint, creating a deployment and executing a workflow.
```
Comment (dankilman) [inline content/cli/install.md]: it is somewhat confusing mentioning the you don't go through the process. While you don't *manually* go through the process, this is exactly the process that is automated for you

File: content/cli/plugins.md [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+
+### upload
+
+Usage: `cfy plugins upload -p PLUGIN_FILE`
+
+Upload a plugin to the manager.
+
+#### Required flags
+
+* `-p, --plugin-path=PLUGIN_FILE` - The path to the Cloudify plugin (`.wgn` file) you would like to upload. Note that only `tar.gz` formatted wagons are allowed.
```
Comment (dankilman) [inline content/cli/plugins.md]: The term `tar.gz` formatted wagons is somewhat confusing. I know what you mean here, but a user may be confused as to whether wgn is expected? tar.gz? etc...

File: content/cli/uninstall.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
+---
+layout: bt_wiki
+title: uninstall
+category: Docs
+draft: false
+abstract: Cloudify's Command-Line Interface
+weight: 230
+---
+
+The `cfy uninstall` command is used to uninstall an application using a Cloudify manager without going through the process of executing working, deleting a deployment and deleting a blueprint.
```
Comment (dankilman) [inline content/cli/uninstall.md]: same comment as in `install`

**Pull request merged**